src/consensus/params.h

```diff
@@ -18,21 +18,24 @@ namespace Consensus {
 /**
  * A buried deployment is one where the height of the activation has been hardcoded into
  * the client implementation long after the consensus change has activated. See BIP 90.
+ * Consensus changes for which the new rules enforced from genesis are not listed here.
```

Comment:
Should this be like https://github.com/bitcoin/bitcoin/pull/26201/files#diff-decae4be02fb8a47ab4557fe74a9cb853bdfa3ec0fa1b515c0a1e5de91f4ad0bR1418?

```suggestion
 * Consensus changes for which the new rules are enforced from genesis are not listed here.
```

Reply:
Done